cmd/main.go

```diff
@@ -453,12 +453,21 @@ var appCmds = []cli.Command{
 	updateCmd,
 }
 
+func printMCVersion(c *cli.Context) {
+	fmt.Fprintf(c.App.Writer, "%s version %s (%s/%s)\n", c.App.Name, c.App.Version, runtime.GOOS, runtime.GOARCH)
+	fmt.Fprintf(c.App.Writer, "Copyright (c) 2015-2022 MinIO, Inc.\n")
```

Comment:
The end year should be dynamic @vadmeste ?

Reply:
> The end year should be dynamic @vadmeste ?

yes, working on it

Reply:
> yes, working on it

Done